src/messages/store/publish.ts

```diff
@@ -25,7 +25,7 @@ type StoreContent = BaseContent & {
  *
  * @param spc The configuration used to publish a store message.
  */
-export async function Publish(spc: StorePublishConfiguration): Promise<string> {
+export async function Publish(spc: StorePublishConfiguration): Promise<BaseMessage> {
```

Comment:
This method should return the content hash.

Reply:
Fixed in https://github.com/aleph-im/aleph-sdk-ts/pull/3/commits/b34344111a7430ecff5164032b9351736a298a7d